### PR TITLE
Improve patent text on notes

### LINKED ISSUE #1443: Upcoming update to WG notes patent policy text
@plehegar and @wseltzer agreed to simplify the patent policy text for **WG Notes** (other types of documents remain unchanged).
After https://github.com/w3c/specberus/pull/707 is merged, the markup should simply be:
```html
<p data-deliverer="@@Working Group ID@@">
  This document was produced by a group operating under the  <a href="https://www.w3.org/Consortium/Patent-Policy/"><abbr title="World Wide Web Consortium">W3C</abbr> Patent Policy</a>.
</p>
```

Note: we are adding a requirement on the `data-deliverer` attribute which should list the WG IDs which was already in one of the previous patent policy link.

Once we are certain respec and bikeshed have been updated, we will update pubrules.

### PR DESCRIPTION
**Pull request by marcoscaceres**
* closes #1443

### CODE REVIEW

**Comment by marcoscaceres** [comment]
@deniak, let me know if this looks right.

File: src/w3c/headers.js [hunk 1]
```diff
@@ -634,11 +634,23 @@ export function run(conf, doc, cb) {
   conf.isPR = conf.specStatus === "PR";
   conf.isPER = conf.specStatus === "PER";
   conf.isMO = conf.specStatus === "MO";
+  conf.isNote = conf.specStatus === "NOTE";
```
Comment (deniak) [inline src/w3c/headers.js]: Looking at the [documentation](https://github.com/w3c/respec/wiki/specStatus), shouldn't the status be `WG-NOTE`?

File: tests/spec/w3c/headers-spec.js [hunk 2]
```diff
@@ -521,6 +521,52 @@ describe("W3C — Headers", function() {
...
+      const elem = doc.querySelector("p[data-deliverer]");
+      expect(isNote).toBe(true);
+      expect(wgId).toEqual("123456");
+      expect(elem).toBeTruthy();
+      expect(elem.dataset.deliverer).toEqual("123456");
+    });
+    it("gracefully handles missing wgPatentURI", async () => {
+      const ops = makeStandardOps();
+      const newProps = {
+        specStatus: "NOTE",
```
Comment (deniak) [inline tests/spec/w3c/headers-spec.js]: `WG-NOTE` instead of `NOTE`?

File: tests/spec/w3c/headers-spec.js [hunk 3]
```diff
@@ -521,6 +521,52 @@ describe("W3C — Headers", function() {
     });
   });
 
+  describe("wgId, data-deliverer, and isNote", () => {
+    it("derives the wgId from wgPatentURI and adds data-deliverer", async () => {
+      const ops = makeStandardOps();
+      const newProps = {
+        wgPatentURI: "https://www.w3.org/pp-impl/123456/status",
+        specStatus: "NOTE",
```
Comment (deniak) [inline tests/spec/w3c/headers-spec.js]: `WG-NOTE` instead of `NOTE`?

File: src/w3c/headers.js (hunk 1 above)
Comment (marcoscaceres) [reply src/w3c/headers.js]: You are correct, there is also FPWD-NOTE, it seems. I still don't know why we have "NOTE"? It seems to be "a thing" - it has an associated W3C Style? Should we kill that?

File: src/w3c/headers.js [hunk 4]
```diff
@@ -634,11 +634,23 @@ export function run(conf, doc, cb) {
   conf.isPR = conf.specStatus === "PR";
   conf.isPER = conf.specStatus === "PER";
   conf.isMO = conf.specStatus === "MO";
+  conf.isNote = ["FPWD-NOTE", "WG-NOTE"].includes(conf.specStatus);
```
Comment (marcoscaceres) [inline src/w3c/headers.js]: @deniak, let me know if "FPWD-NOTE" applies or not. I'm was unsure.

Comment (deniak) [reply src/w3c/headers.js]: Yes, I think that applies too!

**Comment by marcoscaceres** [comment]
Will release later today.

**Pull request merged**